src/main/java/seedu/address/model/person/Attendance.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a Student's Attendance.
+ */
+public class Attendance {
+    private final LocalDateTime date;
+    private final Boolean hasAttended;
```

Comment:
I suggest this is made `public` to remain consistent with the rest of the codebase (see other classes in the same directory).